doc/classes/String.xml

```diff
@@ -720,6 +720,13 @@
 				Replaces all [b]case-insensitive[/b] occurrences of [param what] inside the string with the given [param forwhat].
 			</description>
 		</method>
+		<method name="resize">
+			<return type="int" />
+			<param index="0" name="size" type="int" />
+			<description>
+				Resizes this string to the new size. The size should be one more than desired string length (in order to accommodate the null terminator).
```

Comment:
Doesn't this make the String data mutable?

In most programming languages, Strings are immutable. I think this is true for all String methods in GDScript except for this one. Wouldn't it be better to return a copy that's resized?